### PR TITLE
Fixes for mount and loadkeys

### PR DESCRIPTION
**Pull request by yizhao1**

### CODE REVIEW

File: policy/modules/system/mount.te [hunk 1]
```diff
@@ -100,6 +100,7 @@ files_dontaudit_write_all_mountpoints(mount_t)
 files_dontaudit_setattr_all_mountpoints(mount_t)
 
 fs_getattr_all_fs(mount_t)
+fs_getattr_all_dirs(mount_t)
```
Comment (pebenito) [inline policy/modules/system/mount.te:103]: Why is access beyond mountpoints needed?

Comment (yizhao1) [reply policy/modules/system/mount.te:103]: Without this, the tracefs and configfs can not be mounted at boot.
$ mount |grep -E 'tracefs|configfs'

After add this rule:
$ mount |grep -E 'tracefs|configfs'
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime,seclabel)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)

Comment (pebenito) [reply policy/modules/system/mount.te:103]: It still seems like too much access to add. What are the denial message(s)?

Comment (yizhao1) [reply policy/modules/system/mount.te:103]: $ mount -t tracefs nodev /sys/kernel/tracing
mount: /sys/kernel/tracing: filesystem was mounted, but any subsequent operation failed: Operation not permitted.

$ mount -t configfs none /sys/kernel/config
mount: /sys/kernel/config: filesystem was mounted, but any subsequent operation failed: Operation not permitted.

Comment (pebenito) [reply policy/modules/system/mount.te:103]: What are the audit messages for these denials?

Comment (yizhao1) [reply policy/modules/system/mount.te:103]: type=AVC msg=audit(1695718173.815:89): avc:  denied  { getattr } for  pid=769 comm="mount" path="/" dev="tracefs" ino=1 scontext=root:sysadm_r:mount_t:s0-s15:c0.c1023 tcontext=system_u:object_r:tracefs_t:s0 tclass=dir permissive=0

type=SYSCALL msg=audit(1695718173.815:89): arch=c000003e syscall=332 success=no exit=-13 a0=4 a1=7f47b2629df4 a2=1000 a3=1000 items=0 ppid=477 pid=769 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=ttyS0 ses=4294967295 comm="mount" exe="/usr/bin/mount.util-linux" subj=root:sysadm_r:mount_t:s0-s15:c0.c1023 key=(null)ARCH=x86_64 SYSCALL=statx AUID="unset" UID="root" GID="root" EUID="root" SUID="root" FSUID="root" EGID="root" SGID="root" FSGID="root"


type=AVC msg=audit(1695718194.649:90): avc:  denied  { getattr } for  pid=788 comm="mount" path="/" dev="configfs" ino=19751 scontext=root:sysadm_r:mount_t:s0-s15:c0.c1023 tcontext=system_u:object_r:configfs_t:s0 tclass=dir permissive=0

type=SYSCALL msg=audit(1695718194.649:90): arch=c000003e syscall=332 success=no exit=-13 a0=4 a1=7f38dabebdf4 a2=1000 a3=1000 items=0 ppid=477 pid=788 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=ttyS0 ses=4294967295 comm="mount" exe="/usr/bin/mount.util-linux" subj=root:sysadm_r:mount_t:s0-s15:c0.c1023 key=(null)ARCH=x86_64 SYSCALL=statx AUID="unset" UID="root" GID="root" EUID="root" SUID="root" FSUID="root" EGID="root" SGID="root" FSGID="root"

**Review by pebenito — APPROVED** [review]

**Pull request merged**